packages/plugin-vue/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019-present, Yuxi (Evan) You and contributors
+Copyright (c) 2019-present, Yuxi (Evan) You and Vite contributors
```

Comment:
I don't think this change is necessary.

Reply:
Looking at the other subprojects under the `packages` folder of this project, almost every `LICENSE` is:

> Copyright (c) 2019-present, Yuxi (Evan) You and Vite contributors

Such as:

* https://github.com/vitejs/vite/blob/main/packages/create-vite/LICENSE
* https://github.com/vitejs/vite/blob/main/packages/plugin-react/LICENSE
* https://github.com/vitejs/vite/blob/main/packages/plugin-vue-jsx/LICENSE
* https://github.com/vitejs/vite/blob/main/packages/vite/LICENSE.md

There are two exceptions, one is [`@vitejs/plugin-legacy`](https://github.com/vitejs/vite/tree/main/packages/plugin-legacy), it does not have `LICENSE`, the other is This, it's missing the `Vite` word.

Besides, the main package [`vite`](https://github.com/vitejs/vite/tree/main/packages/vite) uses the MarkDown format of `LICENSE.md`, the others do not.

I think this detail should be unified, so I added the word `Vite` first, and may add `LICENSE` of `plugin-legacy` later, and add or remove the file format of `.md` .

Reply:
Make sense. Thanks for explaining.

Reply:
> ... and may add LICENSE of `plugin-legacy` later, and add or remove the file format of `.md` .

The `plugin-legacy` package has been added.

As for `.md`, I found that the LICENSE of the `vite` package is not the same as other packages, which is reasonable, so I will not modify it.